idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -13,35 +14,39 @@ export default PageView.extend({
     'click [data-hook=uninstall]': 'uninstall'
   },
 
+  bindings: {
+    'model.title': {
+      type: 'text',
+      hook: 'title'
+    },
+
+    'model.enabled': {
+      type: 'toggle',
+      yes: '[data-hook=uninstall]',
+      no: '[data-hook=install]'
+    }
+  },
+
   initialize(opts) {
     this.model = app.experiments.get(opts.slug, 'slug');
   },
 
   render() {
-    this.modified_date = new Date(this.model.modified);
-    this.created_date = new Date(this.model.created);
-
     // TODO: let's not mess with body, if possible
-
-    // TODO:(DJ) enabled property is not set here when navigating
-    // directly to the experiment page. This will not be a problem
-    // when we start persisting state with the User Installation model.
-    if (this.model.enabled) {
-      document.body.classList.add('active');
-    } else {
-      document.body.classList.add('inactive');
-    }
     document.body._id = document.body.id;
     document.body.id = 'idea-view';
 
     PageView.prototype.render.apply(this, arguments);
+
+    this.renderSubview(new DetailsView({
```

Comment:
Hmm, looking at that again, this seems like a problem with the way we're doing the base -> page view subclasses. Might be better to have page view use `renderSubview` instead of declaring `subviews` so we can keep most views simpler.